src/cdc_enumerate.c

```diff
@@ -370,7 +366,7 @@ typedef struct {
     uint8_t data[70];
 } StringDescriptor;
 
-static const char *string_descriptors[] = {0, VENDOR_NAME, PRODUCT_NAME, };
+static const char *string_descriptors[] = {0, VENDOR_NAME, PRODUCT_NAME, "4242"};
```

Comment:
Shouldn't the serial number actually be unique?

Reply:
In theory, I guess, yes. That `"4242"` was added upstream. We could just leave it for now and open an issue, use a random number, or adapt your serial number code.

Reply:
Ok, I'll hunt it down upstream and comment.

Reply:
I meant open our own issue. I'm not sure they care a lot, since it's only an issue if you're plugging in multiple boards at once, right? We could fix it eventually and contribute it to upstream.

Reply:
Not necessarily. I'd worry about browsers and OSes storing settings based on it and applying them to other devices that are plugged in later.